web_src/js/components/DashboardRepoList.vue

```diff
@@ -362,10 +363,25 @@ export default sfc; // activate the IDE's Vue plugin
           <svg-icon name="octicon-plus"/>
         </a>
       </h4>
-      <div class="ui attached segment repos-search">
+      <div v-if="isLoading" class="ui attached segment" :class="{'is-loading': isLoading}"/>
+      <div v-if="!isLoading && !repos.length" class="ui attached segment empty-placeholder">
+        <svg-icon name="octicon-no-entry" :size="48" class-name="repo-list-icon"/>
+        <h2>{{ textNoRepo }}</h2>
+        <p>
+          <a :href="subUrl + '/repo/create'">
+            <svg-icon name="octicon-plus" :size="16" class-name="repo-list-icon"/> {{ textNewRepo }}
+          </a>
+        </p>
+        <p v-if="canCreateMigrations">
+          <a :href="subUrl + '/repo/migrate'">
+            <svg-icon name="octicon-repo-push" :size="16" class-name="repo-list-icon"/> {{ textNewMigrate }}
+          </a>
+        </p>
+      </div>
+      <div v-if="repos.length" class="ui attached segment repos-search">
         <div class="ui small fluid action left icon input">
           <input type="search" spellcheck="false" maxlength="255" @input="changeReposFilter(reposFilter)" v-model="searchQuery" ref="search" @keydown="reposFilterKeyControl" :placeholder="textSearchRepos">
-          <i class="icon loading-icon-3px" :class="{'is-loading': isLoading}"><svg-icon name="octicon-search" :size="16"/></i>
+          <i class="icon loading-icon-3px"><svg-icon name="octicon-search" :size="16"/></i>
```

Comment:
Are you sure this change is right? Especially when users are typing in the input to search?